perf_bench_test.go

```diff
@@ -24,127 +24,344 @@ package main
 
 import (
 	"context"
+	"encoding/csv"
 	"flag"
+	"os"
 	"strconv"
+	"strings"
 	"sync"
+	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/ease-lab/vhive/profile"
 	log "github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	isColdStart     = flag.Bool("coldStart", false, "Profile cold starts (default is false)")
-	vmNum           = flag.Int("vm", 2, "The number of VMs")
-	targetReqPerSec = flag.Int("requestPerSec", 2, "The target number of requests per second")
-	executionTime   = flag.Int("executionTime", 2, "The execution time of the benchmark in seconds")
+	// shared arguments
+	injectDuration = flag.Float64("injectTimeDuration", 1, "The total time in seconds for injecting requests each round")
```

Comment:
you should not profile for less than 10 seconds.

Reply:
Done